### ISSUE TITLE
Warn instead of error out for missing .env file of PORT_GIT_DEPS programs?

### ISSUE THREAD

**Issue by fanbo-meng** [ZOSOpenTools/utils#22]
For the programs specified for `PORT_GIT_DEPS`, e.g make, git, they are already pre-installed somewhere on the machine I'm using, but I have to manually create empty `.env` files under `${HOME}/zot/prod/${dep}/.env` in order for the build.sh script to keep going and not report error during `checkDeps()` step. This gives a bit inconvenience and confusion.

**Comment by IgorTodorovskiIBM** [comment]
One other alternative would be to check if the dependency executable can be found in user's PATH (using `type`) and if it is, do not emit the error

**Comment by MikeFultonDev** [comment]
For git in particular, we do need a particular level of git to operate. So if we do this, we should also add a check of the git version to make sure it is 'new' enough.

**Comment by MikeFultonDev** [comment]
I did add 'git' and 'xlclang' version checks to the code. @fanbo-meng I am thinking once we get 'git' built from open source this problem will 'solve itself' since it will then have a .env file?

**Comment by fanbo-meng** [comment]
yeah I guess if we always build PORT_GIT_DEPS programs like git from open source and use it in building the actual program then .env would always be there.

**Comment by MikeFultonDev** [comment]
@fanbo-meng ok to close?
